source/adaptive_load/adaptive_load_controller_impl.cc

```diff
@@ -172,6 +172,18 @@ absl::StatusOr<AdaptiveLoadSessionOutput> AdaptiveLoadControllerImpl::PerformAda
     BenchmarkResult result = result_or.value();
     *output.mutable_adjusting_stage_results()->Add() = result;
 
+    if (spec.has_benchmark_cooldown_duration()) {
+      ENVOY_LOG_MISC(info, "Cooling down before the next benchmark for duration: {}",
+                     spec.benchmark_cooldown_duration());
+      absl::StatusOr<uint64_t> sleep_time_ms =
+          Envoy::Protobuf::util::TimeUtil::DurationToMilliseconds(
+              spec.benchmark_cooldown_duration());
+      if (!sleep_time_ms.ok()) {
+        return sleep_time_ms.status();
+      }
```

Comment:
I think `StatusOr` here is no longer relevant, right?

Reply:
Done in b4c29db.